docs/topics/mta-web-adding-applications.adoc

```diff
@@ -4,51 +4,52 @@
 
 :_content-type: PROCEDURE
 [id="mta-web-adding-applications_{context}"]
-= Adding an application
+= Adding a new application
 
-You can add an application to the *Application Inventory* for assessment and analysis.
+You can add a new application to the *Application Inventory* for subsequent assessment and analysis.
 
 [TIP]
 ====
 Before creating an application, set up business services, check tags and tag categories, and create additions as needed.
 ====
 
+.Prerequisites
+
+* You are logged in to an {ProductShortName} server.
+
 .Procedure
 
 . In the *Migration* view, click *Application Inventory*.
 . Click *Create new*.
-. Under *Basic information*, enter the following information or choose from a list:
-
-* Name
-* Description (optional)
-* Business service (optional)
-* Manual tags (optional, one or more)
-* Owner (optional)
-* Contributors (optional, one or more)
-* Comments (optional)
-
-. Click the arrow to the left of *Source Code*.
-+
+. Under *Basic information*, enter the following fields:
+* *Name*:A unique name for the new application.  
+* *Description*: A short description of the application (optional).
+* *Business service*: A purpose of the application (optional).
+* *Manual tags*: Software tags that characterize the application (optional, one or more).
+* *Owner*: A registered software owner from the drop-down list (optional). 
+* *Contributors*: Contributors from the drop-down list (optional, one or more).
+* *Comments*: Relevant comments on the migrated application (optional).
```

Comment:
*nitpick:* I don't know if the word "migrated" fit here, the user creates the app to start the migration process, so it is not migrated when it is created.

Reply:
@abrugaro, shall I then just remove "migrated" and say just "Relevant comments on the application"? For me it seems clear enough

Reply:
Yeah @mpershina, that should be enough, thanks!